src/etc/inc/gwlb.inc

```diff
@@ -1343,7 +1343,12 @@ function get_gwgroup_members_inner($group, $gateways_status, $gateways_arr, $vip
 						$groupmember = array();
 						$groupmember['gw'] = $member;
 						$groupmember['int'] = $int;
-						$groupmember['gwip'] = $gatewayip;
+						if(is_ipaddrv6($gatewayip) && is_linklocal($gatewayip))
```

Comment:
Add a space after `if` ([ref](https://docs.netgate.com/pfsense/en/latest/references/developer-style-guide.html#indent-style))

Reply:
Fixed in 671fd0d